src/views/BuyGlp/BuyGlp.js

```diff
@@ -9,7 +9,7 @@ import "./BuyGlp.css";
 import { useChainId } from "../../Helpers";
 import { getNativeToken } from "../../data/Tokens";
 
-import { Trans } from '@lingui/macro'
+import { Trans } from "@lingui/macro";
```

Comment:
wondering why prettier updates these files only now? we haven't changed formatting rules/styleguide, right?